packages/authentication/src/definitions.ts

```diff
@@ -116,6 +132,16 @@ export interface FirebaseAuthenticationPlugin {
   signInWithEmailAndPassword(
     options: SignInWithEmailAndPasswordOptions,
   ): Promise<SignInResult>;
+  /**
+   * Signs in using an email and sign-in email link.
+   *
+   * Both the user's email and the link sent to the user's email address must be provided.
```

Comment:
Since the methods no longer share the interface, you can force the parameter and remove this comment.